.circleci/config.yml

```diff
@@ -739,3 +741,16 @@ commands:
                   command: |
                     export TRAVIS_BRANCH=${CIRCLE_BRANCH}
                     scripts/travis/test_release.sh
+
+  cleanup_workspace:
+    description: Cleanup old build artifacts (especially for persistent self-hosted runners)
+    parameters:
+      build_dir:
+        type: string
+        default: << pipeline.parameters.build_dir >>
+    steps:
+        - run:
+            name: Cleanup Binaries
+            command: |
+              rm -rf /Users/circleci/node_pkg/*
+              rm -rf << parameters.build_dir >>/*
```

Comment:
there is also a `prepare_build_dir` that already does some cleanup. the directory /Users/circleci/node_pkg may only be relevant on macos. you may be able to use `$HOME` so that it is compatible.